src/G2/QuasiQuotes/Internals/G2Rep.hs

```diff
@@ -146,9 +146,11 @@ qqDataConLookupFallBack tyv_n arg_n qqtn qqdc type_nm_qqm dc_nm_qqm tenv
             i = Id n TYPE
 
             t = mkTyFun $ replicate (arg_n + 1) (TyCon n TYPE)
-            t' = foldr TyForAll t (replicate tyv_n i)
+            ty_forall_ids = replicate tyv_n i
+            t' = foldr TyForAll t ty_forall_ids
+            -- might need a double check for this update
```

Comment:
Looks good!

Reply:
So I think those are all the changes I need to make before merging it into the master right?